multimedia/ffmpeg4/Config.in

```diff
@@ -0,0 +1,76 @@
+if PACKAGE_ffmpeg4
+
+menu "Configuration"
+
+comment "Only select one SSL library"
+
+config ffmpeg4_LICENSING
+  bool "See help for licensing information"
+  help
+	Short version:
+	Default license is GNU Lesser General Public License v2.1 or later
+
+	Following changes it to GPL:
+	Building it for x86 due to assembly files
+	Support for libx264
+
+	Following changes it to version 3:
+	Support for mbed TLS
+
+	Following makes it non distributable:
+	Support for FDK-AAC and OpenSSL
```

Comment:
A small, tiny, and completely insignificant note: There's a GPL compatible fork of fdk-aac: https://cgit.freedesktop.org/~wtay/fdk-aac/log/?h=fedora

Reply:
That might be true but I see little to no point in adding unsupported forks.